File: src/Panorama.Core/Scenes/Events/ScenesReceived/ScenesReceivedEventData.cs
using Abp.Application.Services.Dto;
using Panorama.Events.Base;
using Teatro.Shared.Scenes.Dtos;

namespace Panorama.Scenes.Events.ScenesReceived;

public class ScenesReceivedEventData : AppEventData
{
    public PagedResultDto<ViewSceneDto> Data { get; set; }
}

File: src/Panorama.Web.Host/src/shared/service-proxies/scenography/events/ScenesReceivedEventData.ts
import {ViewSceneDto} from "../dtos/ViewSceneDto";
import {PagedResultDto} from "../../common/dtos/PagedResultDto";

export class ScenesReceivedEventData {
    data: PagedResultDto<ViewSceneDto>;
    
    constructor(data: Partial<ScenesReceivedEventData> = {}) {
        this.data = data.data || undefined;
    }
}